### ISSUE TITLE
Add support for multiple instances on a page

### ISSUE THREAD

**Issue by philsherry** [alphagov/accessible-typeahead#67]
We have some pages where the user can add up to ten inputs, via an add/remove component. The current solution doesn't allow for this sort of thing. 

<img width="384" alt="" src="https://cloud.githubusercontent.com/assets/272769/25271723/034f51e8-267d-11e7-8874-0c40d3a21b22.png">

**Comment by edwardhorsford** [comment]
I think this is pretty high priority - there's loads of cases where a service may need more than one of these inputs on a page.

**Comment by tvararu** [comment]
The use case you're listing there might be better served by a `multiselect` feature we're thinking about building later.

> The current solution doesn't allow for this sort of thing.

How is that? The doc pages do have multiple typeaheads all in parallel. For them to work correctly you need each one to have:

- its own `<label>`
- a unique `id` (so that you can hook it up to the label)
- a unique `name` (so the form picks them up separately)*

(I can't confirm for sure if the latter works correctly in a real `<form>` as I haven't done this myself and [we don't have a demo with that yet](https://github.com/alphagov/accessible-typeahead/issues/42))

**Comment by edwardhorsford** [comment]
Sidebar - multiselect is something we could explore, but I'd want to test that it works for users. I have a suspicion that the pattern that phil shows may work better for people lower on the digital literacy scale.

**Comment by tvararu** [comment]
To the best of my knowledge there should be a way to do this (as that's the way the examples page works). Multiselect is a separate issue.

@philsherry please comment / re-open if I misunderstood the bug completely and the examples page solution doesn't work.
